app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="login-container">
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] @GhausAliKhan and @78fatemanazari Great work implementing the logging page, however, the layout doesn't match the [the wireframes from the Sneak Peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md#:~:text=For%20this%20project%20you%20will%20have%20full%20freedom%20in%20terms%20of%20visual%20design%20but%20you%20will%20need%20to%20keep%20the%20following%20wireframes%3A) given, below is the comparison:



Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/GhausAliKhan/Blog-app/assets/61408860/5c22d079-7996-48d7-84e4-119e82e1eb5b) |  ![image](https://github.com/GhausAliKhan/Blog-app/assets/61408860/2d02070b-637f-4f43-ae53-0623982c2caa)